charts/ziti-router/values.yaml

```diff
@@ -151,6 +151,12 @@ image:
   # template).
   # -- deployment container command args and opts
   args: [ 'run', '{{ .Values.configMountDir }}/{{ .Values.configFile }}']
+  # -- additional arguments can be passed directly to the container to modify ziti runtime arguments
+  additionalArgs:
```

Comment:
same two thoughts for router as controller:

- empty list `additionalArgs: []`
- empty dict `env: {}`